eos/form-factors/b-lcdas.cc

```diff
@@ -43,12 +45,10 @@ namespace eos
         if (i != models.end())
         {
             result.reset( i->second(parameters, options) );
-        }
-        else
-        {
-            throw InternalError("Unknown value for 'name' for BMesonLCDAs::make");
+            return result;
         }
 
+        throw InternalError("Unknown B-meson LCDA model: " + name);
```

Comment:
Were you able to check that?

Reply:
After revising: yes. All form factors and LCDAs have been instrumented with a test case ensuring that an exception is raised for nonsense inputs.

Reply:
Excellent, thanks a lot!